app/helpers/form_helper.rb

```diff
@@ -20,6 +20,14 @@ def block_date_field_tag(name, value, options = {})
       form_input_error_messages(name, options)
   end
 
+  def idv_using_mock_vendor
+    if Figaro.env.proofing_vendors
+      Figaro.env.proofing_vendors.split(/\W+/).map(&:to_sym).include?(:mock)
+    else
+      true
```

Comment:
yes, that is the way reek likes it. I don't care for it either, which is why I forgot to do it.

Still, that wouldn't fix this particular warning, since it still means calling `Figaro.env` 2x.